lib/robots/dor_repo/speech_to_text/stt_create.rb

```diff
@@ -11,10 +11,51 @@ def initialize
 
         # available from LyberCore::Robot: druid, bare_druid, workflow_service, object_client, cocina_object, logger
         def perform_work
-          # TODO: do the caption creation by calling out to Whisper
+          # start the caption creation by sending an SQS message to the caption creation service
+
+          send_sqs_message
 
           # Leave this step running until the Whisper monitoring job marks it as complete
           LyberCore::ReturnState.new(status: :noop, note: 'Initiated SpeechToText.')
+        rescue StandardError => e
+          Honeybadger.notify('Problem sending SQS Message to AWS for SpeechToText', context: { druid:, error: e })
+          raise "Error sending SQS message: #{e.message}"
+        end
+
+        private
+
+        def aws_provider
+          @aws_provider ||= Dor::TextExtraction::AwsProvider.new(region: Settings.aws.region, access_key_id: Settings.aws.access_key_id, secret_access_key: Settings.aws.secret_access_key)
+        end
+
+        def send_sqs_message
+          message_body = {
+            id: SecureRandom.uuid,
+            druid:,
+            media:
+          }.merge(whisper_options).to_json
+
+          # Send the message to the SQS queue
+          aws_provider.sqs.send_message({
+                                          queue_url: aws_provider.sqs_todo_queue_url,
+                                          message_body:
+                                        })
+
+          logger.info("Sent SQS message for druid #{druid} to queue #{aws_provider.sqs_todo_queue_url}")
+        end
+
+        def media
+          Dor::TextExtraction::SpeechToText.new(cocina_object:, workflow_context: workflow.context).filenames_to_stt
+        end
```

Comment:
this will reach out to DSA again and ask for the list of available files (the same ones we sent to S3 in the previous workflow step)